package.json

```diff
@@ -76,6 +76,8 @@
     "@ava/typescript": "^1.1.1",
     "@babel/core": "^7.9.6",
     "@octokit/graphql": "2.1.1",
+    "@tsconfig/node16": "^1.0.3",
+    "@tsconfig/node16-strictest": "^1.0.3",
```

Comment:
What are these for?

Reply:
Do we need both "@tsconfig/node16" and "@tsconfig/node16-strictest"?

Reply:
why didn't we have @tsconfig/node14 as dependency?

Reply:
@Jkovarik do you know why we don't need `"@tsconfig/node14` and if it's necessary to add `"@tsconfig/node16-strictest`?

Reply:
@jennyhliu  I don't know off the top of my head -- I think the AC on this ticket was talking about updating lines like 

```
    "target": "ES2019",
```
in the tsconfig.json.    I'm perhaps misunderstanding the question though.

Reply:
I'm assuming this was related to the arch meeting discussion where utilizing a common ts configuration was discussed?

Reply:
There are other places in the baseline have "target": "es2019" and "target": "ES2020", please check if we need to update those.